config/app.yml

```diff
@@ -1,5 +1,5 @@
 shared: &shared
-  version: '2.15.0'
+  version: '2.15.1'
```

Comment:
This is not required. We update the version when publishing a new release

Reply:
I would like to be able to identify the version that I modified for my internal control here. What do you suggest?

Reply:
You can retain the version variable separately on your fork